### DISCUSSION TITLE
aws_sqs: ignore message when processed

### DISCUSSION THREAD

**Discussion by lukasmrtvy** [vectordotdev/vector#21903] [Q&A]
Hey. Is it possible to ignore a message without using `delete_message: true` ?

**Comment by jszwedko** [comment]
No, I don't believe it is possible to ignore a message. The source will attempt to process all messages in the specified queue. If it fails to process the message a certain number of times you can have AWS send it to a dead letter queue (see https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-dead-letter-queues.html).

**Reply by jszwedko** [reply]
You can, of course, consume and discard the message.
